src/Migrations/Version20250327103803.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * BG2-2854 – Remove Gift Aid from one donation
+ */
+final class Version20250327103803 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Remove Gift Aid from one donation';
+    }
+
+    public function up(Schema $schema): void
+    {
+        /**
+         * @var array{uuid: string, transactionId: string}
+         */
+        $idPair = [
+            'uuid' => '01f1edad-1f3b-4441-a21c-6c031eda9ee8',
+            'transactionId' => 'pi_3R5k5KKkGuKkxwBN0SIhyZre',
+        ];
+
+        $this->addSql(<<<EOT
+            UPDATE Donation
+            SET salesforcePushStatus = 'pending-update', giftAid = 0, tipGiftAid = 0
```

Comment:
not essential but probably worth doing from now on:
```suggestion
            SET salesforcePushStatus = 'pending-update', giftAid = 0, tipGiftAid = 0, giftAidRemovedAt = now(),
```

Reply:
Thanks, done in c62c0c571ebd71d42934939ab6b156cbb4ce179c as I wasn't sure if the engine would deal with an extra comma

Reply:
ah yep no I don't think mysql does allow trailing commas.